src/test/compressor/test_compression_zlib.cc

```diff
@@ -177,6 +177,69 @@ TEST(ZlibCompressor, zlib_isal_compatibility)
   EXPECT_TRUE(exp.contents_equal(after));
 }
 
+TEST(ZlibCompressor, isal_compress_zlib_decompress_random)
```

Comment:
Looks like you're using legacy master fork, compression UTs are a bit different now